src/startupscriptgenerator/src/common/commandLineSwitches.go

```diff
@@ -0,0 +1,16 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import "flag"
+
+func GetManifestDirSwitch() *string {
```

Comment:
1. does this have to be a function? can it just be an exported variable?
2. these are called Flags, not Switches, in the Go jargon - let's keep aligned to it